packages/core/src/fhir-deduplication/resources/observation-shared.ts

```diff
@@ -57,7 +57,8 @@ export function extractCodes(concept: CodeableConcept | undefined): {
           if (
             system.includes("unknown") &&
             code.includes(unknownCoding.code) &&
-            (display === unknownCoding.display || text === unknownCode.text)
+            display === unknownCoding.display &&
```

Comment:
losing good data

Reply:
Sometimes the `display` says "unknown", and the `text` field is missing altogether. This is why I put an `OR` between them. But I'm fine with this for now

Reply:
yah that makes sense